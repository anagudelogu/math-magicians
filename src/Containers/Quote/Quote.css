.quote {
  color: var(--light-gray);
  text-align: center;
  padding: 1rem;
  min-width: 300px;
  height: calc(100vh - 2rem);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  overflow: hidden;
}

.quote__text {
  font-size: 1.1rem;
  line-height: 1.3rem;
  margin-bottom: 0.5rem;
}

.quote__author {
  width: 100%;
  text-align: right;
}

@media screen and (min-width: 768px) {
  .quote {
    max-width: 800px;
  }

  .quote__text {
    font-size: 2rem;
    line-height: 2.5rem;
    margin-bottom: 2rem;
  }

  .quote__author {
    font-size: 1.5rem;
  }
}
