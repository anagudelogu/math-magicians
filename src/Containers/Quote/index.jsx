import React from 'react';
import './Quote.css';

const Quote = () => (
  <div className="quote">
    <p className="quote__text">
      Mathematics is not about numbers, equations, computations, or
      algorithms: it is about understanding.
    </p>
    <p className="quote__author">- William Paul Thurston</p>
  </div>
);

export default Quote;
