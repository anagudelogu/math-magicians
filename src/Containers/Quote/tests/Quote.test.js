import renderer from 'react-test-renderer';
import Quote from '../index';

describe('Quote page', () => {
  it('Renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
