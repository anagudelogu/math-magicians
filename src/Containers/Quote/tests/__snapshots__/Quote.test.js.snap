// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote page Renders correctly 1`] = `
<div
  className="quote"
>
  <p
    className="quote__text"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
  </p>
  <p
    className="quote__author"
  >
    - William Paul Thurston
  </p>
</div>
`;
